cmd/goreleaser/internal/configure.go

```diff
@@ -144,21 +148,29 @@ func Package(dist string) config.NFPM {
 func DockerImages(imagePrefixes, dists []string) (r []config.Docker) {
 	for _, dist := range dists {
 		for _, arch := range Architectures {
-			r = append(r, DockerImage(imagePrefixes, dist, arch))
+			switch arch {
+			case ArmArch:
+				for _, vers := range ArmVersions {
+					r = append(r, DockerImage(imagePrefixes, dist, arch, vers))
+				}
+			default:
+				r = append(r, DockerImage(imagePrefixes, dist, arch, ""))
+			}
```

Comment:
Initially, for including the optional arm version, I thought about creating a new configuration type to default the arm version to empty string. For example:

```golang
type DockerImageConfig struct {
  ImagePrefixes []string
  Dist string
  Arch string
  ArmVersion string
}
```
Then in the future we can add more optional values such as AMD64 versions. However, it seems like that's more complicated for the current usage, that is, in most cases we only pass around 3 values (imagePrefixes, dist and arch). So I opted to add new argument to `DockerImages` instead. Happy to change it any other suggestions/alternatives.